torch/_inductor/lowering.py

```diff
@@ -5930,6 +5931,117 @@ def while_loop(cond_fn, body_fn, operands):
     return list(map(TensorBox.create, result))
 
 
+@register_lowering(torch.ops.higher_order.templated_attention)
+def templated_attention(*args, **kwargs):
+    from torch._prims_common import make_contiguous_strides_for
+    from .ir import (
+        ComputedBuffer,
+        FixedLayout,
+        FlexibleLayout,
+        InputBuffer,
+        StorageBox,
+        TensorBox,
+    )
+
+    query, key, value, subgraph = args
+
+    def create_placeholder(name: str, dtype: torch.dtype) -> InputBuffer:
+        return TensorBox.create(
+            InputBuffer(
+                name,
+                FixedLayout(
+                    query.get_device(),
+                    dtype,
```

Comment:
Does this need to be formatted like this 🤔

Reply:
😭 lintrunner is being particularly ugly here